app/forms/state_file/nc_county_form.rb

```diff
@@ -0,0 +1,14 @@
+module StateFile
+  class NcCountyForm < QuestionsForm
+    set_attributes_for :intake,
+      :residence_county
+
+    validates :residence_county,
+      presence: true,
+      inclusion: { in: StateFileNcIntake::COUNTIES.keys }
```

Comment:
this is also a question that I generally have about validations -- if we have validations like this on the `Form` and the value eventually gets saved on the given `intake` record in the DB, there is no validation on the record (i.e. you're not adding the same validation on the `residence_county` on the `intake` model), is this our accepted pattern and what would be the reason for not adding that on the intake model?

Reply:
I very much agree. I think with the validators being spread around it's pretty wonky. When I initially wrote this I actually included this validation in the concern and was going to keep the form 'dumb' to leverage the machinery in place to hook into the form objects. This is the first page I've implemented using form objects so I am unwilling to mount a strong case _right now_ but after I have a couple under my belt I would like to suggest something new.

I bet we could write an easy form adapter and then pass the validations in concerns to keep them nice and sliced up/segmented and minimize rewrite as well as reduce risks of the database getting into an inconsistent state. But again, I need more exposure before I can competently argue in any direction.

Reply:
ooh yeah great idea, ty for confirming I'm not alone here :)